### ISSUE TITLE
Content is not displaying.

### ISSUE THREAD

**Issue by ashiq35** [dudaster/ele-custom-skin#626]
Hello,
Thanks for the awesome plugin, this plugin really makes our life easier.

I am facing a problem recently, the page that contains the custom skin loop template is displaying correctly and the pages/ elementor template where the custom skin loop is not used are completely blank pages. Only the Eelementor header and footer are displayed, the main content area is missing. If I disable your plugin then everything is working fine but I need your plugin because I decorate some pages by using your plugin.

I am using the free version of your plugin, I had this issue last week (22nd November) and it was solved automatically. And today(29th November) the problem appears again. All the plugins and everything are updated in my area. 

WP Version: 5.8.2
Elementor: 3.4.8
Elementor pro: 3.5.2
Ele Custom Skin: 3.1.4

Regards,
Ashiq

**Comment by clancel** [comment]
I had the same problem.  What I did was I created a Single Page template and displayed the conditions to a page in order for the content to appear.  I first went to a page, saved it as a Page template.  I then created a single page and added the Page to the Single Page.  Afterwards, I displayed the conditions to that page.  Of course, I would prefer to create and edit pages directly through Pages without going this route.

**Comment by ElColo** [comment]
We just had the same issue happen. Wherever a custom template is, it breaks the page. When I try to edit the broken page in Elementor, I get the warning "The Post Content Widget was not found in youur template". See screenshot
<img width="393" alt="Screen Shot 2021-12-01 at 3 40 18 PM" src="https://user-images.githubusercontent.com/7650163/144325960-4176033c-77c8-4a72-82c9-062d3aa4ecb9.png">
.

**Comment by clancel** [comment]
With Elementor Custom Skin activated, you can only edit from a template and not form a page.  I would save the page into a template and create a single page template instead.  You can insert the saved Page into the Single Page template and display the condition to that page.  This route worked for me although I would prefer to edit directly from Pages without going through the template route.

**Comment by robhaskin** [comment]
I am having the same problem - When I disable ele custom skin everything works fine but when activated I only get the header and footer and all content in between disappears. Please fix. Thank you!!

**Comment by inpublica** [comment]
Hi, i have problems too. When I have installed the plugin, the website only works to a limited extent. when editing the page i get the same error message.

**Comment by ElColo** [comment]
I was able to fix the problem by creating a single-page template. We use the Hello theme and the theme builder, and we didn't have a single-page template setup. It seems Custom Skins requires it. We created one, assigned it to a random page, and everything worked again.

**Comment by ElColo** [comment]
> With Elementor Custom Skin activated, you can only edit from a template and not form a page. I would save the page into a template and create a single page template instead. You can insert the saved Page into the Single Page template and display the condition to that page. This route worked for me although I would prefer to edit directly from Pages without going through the template route.

@clancel Could it be that it works now because a single-page template now exists? Have you tried doing it the normal way again since you now have a single-page template? All I need to do was just create one single-page template and everything worked, regardless of whether or not that template is used for a given page.

**Comment by robhaskin** [comment]
> I am having the same problem - When I disable ele custom skin everything works fine but when activated I only get the header and footer and all content in between disappears. Please fix. Thank you!!

@ElColo i created a single page template and assigned it to nothing. It totally worked for me. Thanks for the solution!!!!

**Comment by inpublica** [comment]
... this hint only helped me for a short time. As soon as I change something on a template (e.g. header template) and save it again, the entire content is no longer visible/displaying. It is not a conflict with another plugin either, I have already deactivated all other plugins. My guess is that the problem is related to a recent Elementor update. However, I don't know what to do anymore.

**Comment by dudaster** [comment]
this is wierd.... can you provide a staging website? If you can please deliver the login on website chat.

**Comment by robhaskin** [comment]
> ... this hint only helped me for a short time. As soon as I change something on a template (e.g. header template) and save it again, the entire content is no longer visible/displaying. It is not a conflict with another plugin either, I have already deactivated all other plugins. My guess is that the problem is related to a recent Elementor update. However, I don't know what to do anymore.

Correct. As soon as I opened the blog single template and made an edit and saved it - same thing happened - all content disappeared until I turned ele custom skin off

**Comment by robhaskin** [comment]
I just posted some login details for my staging server so you can see what
is happening

**Comment by robhaskin** [comment]
In your chat on the website

**Comment by RKRJOE** [comment]
I'm having the same issue, but only on mobile.

**Comment by robhaskin** [comment]
Here is the only way I can get it to work on my site: 
1. If I edit any loop or single-page templates in theme builder with Ele custom skin plugin turned on - all of the content disappears on all pages.
2. Deactivate Ele and all content returns
3. Edit any single template and save.
4. Reactivate Ele and content is working again.
5. Giant pain to have to do this over and over to edit.
Is there going to be a fix for this? Otherwise, I have to find a new plugin to create loops with.

**Comment by inpublica** [comment]
... Thank you for the tip. I did it like that and the site works again. :-)

**Comment by wpcoaching** [comment]
> Here is the only way I can get it to work on my site:
> 
> 1. If I edit any loop or single-page templates in theme builder with Ele custom skin plugin turned on - all of the content disappears on all pages.
> 2. Deactivate Ele and all content returns
> 3. Edit any single template and save.
> 4. Reactivate Ele and content is working again.
> 5. Giant pain to have to do this over and over to edit.
>    Is there going to be a fix for this? Otherwise, I have to find a new plugin to create loops with.

thanks. this is working. But I hope Dudaster will repair this bug very soon.

**Comment by c24o** [comment]
> Here is the only way I can get it to work on my site:
> 
>     1. If I edit any loop or single-page templates in theme builder with Ele custom skin plugin turned on - all of the content disappears on all pages.
> 
>     2. Deactivate Ele and all content returns
> 
>     3. Edit any single template and save.
> 
>     4. Reactivate Ele and content is working again.
> 
>     5. Giant pain to have to do this over and over to edit.
>        Is there going to be a fix for this? Otherwise, I have to find a new plugin to create loops with.

We are using the pro version and we have the same problem. There are multiple people editing the website and they don't understand the problem neither what they need to do to fix it, at the end the website is broken frequently.

Hopefully the bug is fixed soon.

**Comment by rrodkin** [comment]
+1 on this issue, though my 'fix' for now was to just rollback Elementor Pro to my previous version.  Just found this thread so have yet to try the workarounds noted above.

Hope Dudaster comes back with a solution soon.

**Comment by richardwo** [comment]
+1 I'm also having that issue with v3.1.4 and v3.1.5. If I edit a certain archive template then Elementor content stops showing on the home page. If I try to edit the home page in Elementor I get the message:
"Sorry, the content area was not found in your page.
You must call 'the_content' function in the current template,
in order for Elementor to work on this page".
Restoring a revision of the home page makes no difference, I have to restore the database from a backup to get the home page working again. If I repeat this editing process with v3.1.3 and same versions of all the other plugins the home page is fine.
